pinot-core/src/main/java/org/apache/pinot/core/indexsegment/immutable/ImmutableSegmentLoader.java

```diff
@@ -39,13 +41,13 @@
 import org.apache.pinot.core.segment.index.loader.SegmentPreProcessor;
 import org.apache.pinot.core.segment.store.SegmentDirectory;
 import org.apache.pinot.core.segment.store.SegmentDirectoryPaths;
-import org.apache.pinot.core.segment.virtualcolumn.VirtualColumnContext;
-import org.apache.pinot.core.segment.virtualcolumn.VirtualColumnProvider;
-import org.apache.pinot.core.segment.virtualcolumn.VirtualColumnProviderFactory;
 import org.apache.pinot.core.startree.v2.store.StarTreeIndexContainer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.pinot.common.utils.CommonConstants.Schema.VIRTUAL_COLUMN_HOSTNAME;
```

Comment:
Let's remove the static imports from all the files for the sake of readability. Instead use `Schema.VIRTUAL_COLUMN_HOSTNAME` etc, or better `VirtualColumn.HOSTNAME, VirtualColumn.SEGMENT_NAME` etc.